app/indexers/identifiable_indexer.rb

```diff
@@ -48,7 +50,7 @@ def distinct_current_catalog_types
             .map(&:catalog)
             .uniq
             .sort
-            .reject { |catalog_type| catalog_type.include?('previous') }
+            .reject { |catalog_type| DEPRECATED_CATALOGS.any? { |catalog| catalog_type.include?(catalog) } }
```

Comment:
There are four possible values we can have in this field:

1. `folio`
2. `previous folio`
3. `symphony`
4. `previous symphony`

And the code as currently written rejects 2-4 and allows 1. Which is good! Given "previous" isn't a catalog that's been deprecated, I wonder if we might consider an even more direct approach here:

```suggestion
            .select { |catalog_type| catalog_type == CURRENT_CATALOG_TYPE }
```

Which selects 1 without caring about 2-4, which means less baggage to hang onto.